_studio/mfx_lib/decode/h264/src/mfx_h264_dec_decode.cpp

```diff
@@ -644,6 +644,13 @@ mfxTaskThreadingPolicy VideoDECODEH264::GetThreadingPolicy(void)
     return MFX_TASK_THREADING_SHARED;
 }
 
+mfxStatus VideoDECODEH264::GetThreadNum(mfxU32& threadNum)
+{
+    MFX_CHECK(m_isInit, MFX_ERR_NOT_INITIALIZED);
```

Comment:
hm, that's why it should return status

Reply:
Yes. Unfortunately we can't provide exact number of threads for our legacy components at all times.